.mockery.yaml

```diff
@@ -0,0 +1,11 @@
+with-expecter: false
+inpackage: True
+dir: "./pkg/tangy"
+mockname: "Mock{{.InterfaceName}}"
+outpkg: "{{.PackageName}}"
+filename: "{{.InterfaceName}}_mock.go"
+all: True
```

Comment:
to here I would add `disable-version-string: True`, so we don't have a diff if only the version of mockery changes

Reply:
NOICE! 
Will do.